Makefile

```diff
@@ -7,7 +7,7 @@ CFLAGS+=-I$(INCDIR)
 SRCS=$(wildcard $(SRCDIR)/*.c)
 OBJS=$(patsubst $(SRCDIR)/%.c,$(OBJDIR)/%.o,$(SRCS))
 
-CFLAGS+=-O2 -Wall -std=c99
+CFLAGS+=-O2 -Wall -std=c99 `pkg-config --cflags --libs gtk+-3.0`
```

Comment:
Yes sure, but I only use that part ( https://developer.gnome.org/gdk-pixbuf/2.30/ ). Feel free to revert my push if it was not the aim of the project but it also fix some errors like this https://github.com/emgram769/lighthouse/issues/33 .